__tests__/profileController.test.ts

```diff
@@ -99,9 +99,30 @@ describe('Profile Controller Tests', () => {
       mockRequest = {
         params: { userID: '65117c94f000c9930ef5c0ee' },
         body: {
-          firstName: 'Bobby',
-          lastName: 'Orr',
+          availabilityForNetworking: true,
+          careerInformation: {
+            currentPosition: {
+              company: 'CodeHammers',
+              title: 'Developer',
+            },
+            pastPositions: [
+              {
+                company: 'CodeHammers',
+                title: 'Junior Developer',
+                startDate: '2020-01-01',
+                endDate: '2021-01-01',
+              },
+            ],
+          },
+          cohort: 'ECRI-TEST',
           email: 'test@test.com',
+          gitHubProfile: 'Ghub',
+          linkedInProfile: 'Lin',
+          nickName: 'Johnny',
+          personalBio: 'I love dogs!',
+          skills: ['Javascript', 'Typescript', 'React', 'Nodejs'],
+          socialMediaLinks: 'SMlinks',
```

Comment:
more of a future ticket thing, but if we refactor these tests to use supertest, the mock request bodies will have to match our Profile schema